src/services/static-data-client.ts

```diff
@@ -26,4 +27,31 @@ export class HttpStaticDataClient implements StaticDataClient {
       return { success: false, error: error };
     }
   }
+
+  public async getStatesData(country: string, language: string): Promise<any> {
+    try {
+      const response = await this.axios.request({
+        method: 'GET',
+        url: this.baseUrl + `/${language === 'es' ? 'states-es-v1' : 'states-en-v1'}.json`,
+        headers: { 'Cache-Control': 'no-store,max-age=0' },
```

Comment:
As we talked this should not be neccesary right now, we will see in the future.

Reply:
Done